code/modules/vehicles/_vehicle.dm

```diff
@@ -56,7 +58,7 @@
 			. += "<span class='warning'>It's falling apart!</span>"
 
 /obj/vehicle/proc/is_key(obj/item/I)
-	return I? (key_type_exact? (I.type == key_type) : istype(I, key_type)) : FALSE
+	return istype(I, key_type)
```

Comment:
This would benefit from becoming a define, not to temp future coders to redefine it.
Death to `is_thing()` procs.
It's just a suggestion, don't do it if you don't feel like it.

Reply:
It's not checking if it's an /obj/item/key, this is checking if it's a key that works with this specific vehicle